app/views/dashboard/_status_links.html.erb

```diff
@@ -1,28 +1,30 @@
+<% origin = @data.settings[:origin] %>
+
 <%= searchable_links _('Hosts that had performed modifications without error'),
-                     "last_report > \"#{Setting[:puppet_interval] + Setting[:outofsync_interval]} minutes ago\" and (status.applied > 0 or status.restarted > 0) and (status.failed = 0)",
+                     "origin = #{origin} and last_report > \"#{Setting[:"#{origin.downcase}_interval"]} minutes ago\" and (status.applied > 0 or status.restarted > 0) and (status.failed = 0)",
```

Comment:
fails if this origin is nil (plugin does not define setting), perhaps we should define origin with fallback to empty string at least